### DISCUSSION TITLE
Maximizing Window Width

### DISCUSSION THREAD

**Discussion by SetKu** [rxhanson/Rectangle#676] [Q&A]
Hello rxhanson,

Is it possible within your app to maximize the width of a window?

I've noticed Rectangle has the capability to maximize the height of a window, however, I'd like to be able to maximize the width of a window to fit in the space left the window I have on the right of my screen which isn't a default ratio in the app.

Is this possible or too much to ask to implement?

Sincerely, SetKu.

**Comment by rxhanson** [comment]
I don't have a desire to add in more window commands to the UI, but would consider a pull request for this one as long as it is configured entirely via Terminal command. If anyone wants to give it a go, let me know and I can give some details on where to start.
